install.sh

```diff
@@ -368,7 +390,11 @@ function create_users_and_groups {
 	if [ $NO_DOCKER_GRP_REQUESTED -ne 1 ]; then
```

Comment:
I think we may be able to improve the logic here a bit. e.g. harden it by flipping the condition, so that the lack of this variable will not inadvertently add the user to the `docker` group?

Reply:
Isn't this the default behavior of the installer?  If `--no-docker-grp` is passed, the installer will add the user to `docker` group.

Reply:
if `--no-docker-grp` is passed, the installer will NOT add the user to the `docker` group. I'm arguing we should flip that.

Reply:
Sorry, I missed a negation to my previous comment, edited.

Reply:
I think I don't understand what are you suggesting to flip.

Reply:
I mean the guard should be `-eq` and the whole check flipped to be more defensive, i.e. that we don't add to the docker group by default and they have to explicitly ask for it.

Reply:
I understand. I would tackle this later and wouldn't change the default behaviour of the installer in this branch since it requires changing installation instructions in few places (i.e. platform).

Reply:
ok